### DISCUSSION TITLE
Should we favour existing tagging for names?

### DISCUSSION THREAD

**Discussion by kymckay** [osmlab/name-suggestion-index#5830] [Q&A] [answered]
Have encountered a few instances of this already so it would be good to have a rule in writing to go by.

Sometimes existing tagging is a `name=` value that's close, but not exactly the same as the name presented by the brand themselves. Examples:
- #2233 (second discussion)
- #2235 (CVS Pharmacy)
- #1413

Personally I think we should be updating/adding entries with the name presented by the brand even if it goes against existing tagging and even if the difference is very minor (as the goal here is to provide "canonical" information).

Any thoughts, suggestions, exceptions, edge cases?

**Comment by Adamant36** [comment]
Which name is connected to the actual tagging numbers? Otherwise, you lose all the ones that are tagged already by changing the name to something not used and iD Editor is creating a new entry (mapping point) in the data base that diverges from the original. Essentially leaving the old one behind. At least thats how I think it would work. 

Also, how would we decide if the name is "wrong" exactly since we dont have ground truth? For example if you look at pictures of CVS stores on Google, in front some say "CVS Pharmacy" while some just say "CVS." So while neither is probably wrong, id say its better to go with the existing value because its established and would therefore have more weight on the ground. People tend to tag what they see how they see it. 

Also, I think there's certain sectors like the petroleum industry, where the name is tagged on the fuel pumps instead of the main convenience store building if there is one, that people will always tag with the name of the oil company instead of the store brand. Since its already tagged on the convenience store and its more about the distributor of the fuel then where particularly you are buying it. It seems like most fuel entries are that way. A lot of the fuel industry is made up of large conglomerates that have a lot of cross over. Fuel is a specific product and not a store type in 99% of cases anyway. So, I think #2233 is sort of different from the CVS example and should probably be handled differently. Not that I know exactly how.

**Comment by kymckay** [comment]
@Adamant36 That's a good point and something else I've been curious about. There's a weird issue when it comes to tagging chains in that the signage doesn't always match the name. The CVS example maybe isn't the best as I've now learned that it's also a store of some kind.

However, one I've come across is Sports Direct. All of the stores (at least here in the UK) have a big sign that says "SportsDirect.com". However, the actual store name is just "Sports Direct" which is what you'd call it in conversation (and what the entry here has it tagged as).

Another example here in Scotland is Clydesdale Bank, often branches just have a sign with "CB" on it, but surely you'd tag it with the full name still because that's what it's actually called (at least, I would).

----
As for the majority tagging question, as I see it this is a source of cannonical information and thus should show preference to correct information over existing information. We may not know ground truth, but when it comes to brands tagging should (for the most part) be consistent everywhere.

To your point about losing the existing tagging, I believe that's where the `match` key comes in. You're still recognising those entities as the same thing, just not the correct tagging.

**Comment by Adamant36** [comment]
After putting some thought into it, I don't think there should be a hard and fast rule when it comes to what name to go with. It might work in some circumstances to go with the existing one and not work in others. So it should depend on the brand. 

Deciding when to not go with the existing name should also be based on more then one variable. I.E. what name is tagged the most, what the brands Wikipedia says (Wikipedia isn't 100% accurate though), and other research. going by existing tagging at least has "ground truth" behind it. Which is something OSM puts more weight into. While this is a canonical database, its first a canonical database in relation to OSM. Otherwise why include the names etc from OSM that are currently being used in the first place?

There really isn't a reason to change the name key from what exists anyway from what I can see. 

for instance check out the entry for Walmart. 

```
"brand": "Sam's Club",
      "brand:wikidata": "Q1972120",
      "brand:wikipedia": "en:Sam's Club",
      "name": "Sam's Club",
      "operator": "Walmart",
      "operator:wikidata": "Q483551",
      "operator:wikipedia": "en:Walmart"

```
That should be enough to cover everything without having to change the name tag. So why change it in the first place? (in CVS's case it should actually be CVS Health (operator)--->CVS Pharmacy (brand. They run regular CVS stores, along with the ones that contain pharmacies)--->CVS (store name and where the match thing would come in with name=CVS_pharmacy. So people searching for both CVS and CVS pharmacy will get the same Wikipedia etc data). So, i don't see why the name would need to be to changed (if "brand" is different from "name" so what? They can be different in real life). Otherwise, why have the "match" exception? Unless I'm misunderstanding the point in it. 

Also, if you create a completely new name/brand entry that isn't based on existing tagging, your essentially creating a whole new name=* tag on the OSM side. Since that's the name/brand people would be tagging things as from then on. So that's something to think about. I don't think the name index should add a bunch of new name/brand entries for things in the OSM database when name tags already exist for them just because Wikipedia might use a different name. Its just redundant, not how people tag, and over complicates things. 

There's also nothing wrong with having multiple entries for the same company if need be. I think in a lot of cases though adding the operator tag to compliment the name and brand would help (or just changing the brand name). Again, it should be based on the individual entry and entitle doing the research first though.  


(maybe we could open a meta issue or something for ones that we do think should changed. In general there's nothing wrong with opening issues for things that we are unsure about and also getting feedback from other contributors/admins about it).

**Comment by Adamant36** [comment]
@bhousel, what do you think about it?

**Comment by bhousel** [comment, marked as answer]
Oh yeah this is something that will probably always be full of interesting edge cases and judgement calls...

@SilentSpike said:
> Personally I think we should be updating/adding entries with the name presented by the brand even if it goes against existing tagging and even if the difference is very minor (as the goal here is to provide "canonical" information).

I agree with this... Here are some guidelines I try to follow:

- We almost always want to use the name that the brand prefers.  So, if "Dunkin' Donuts" is now calling themselves "Dunkin'" then that's the name we should suggest to people, even if they are overwhelmingly already mapped as and signed as "Dunkin' Donuts".  Brands are _fluid_ - they change and that's ok.  Once the `brand:wikidata` tags are more widespread, OSM is in better shape, and we can even encourage mappers to cleanup brand names or do mass renames.
- We also need to pick a name that renders reasonably on a map.  So nothing too long or weird.  
  - "P.F. Chang's" is better than "P. F. Chang's China Bistro"
  - "Fairfield Inn" is better than "Fairfield Inn & Suites by Marriott"
  - (I think we should shorten "Applebee’s Neighborhood Grill & Bar" to simply "Applebee's")
- And it has to be generally recognizable by people
  - "UPS" is more recognizable than "United Parcel Service"
  - but we wouldn't shorten "Dunkin' Donuts" to "DD", even though they write that on the cups.
- "CVS" is a tricky case.  I know they are usually signed as "CVS Pharmacy", and that's technically their brand name too, and in their signage and logo, so that's what I'd go with.

@Adamant36 said:
> I don't think the name index should add a bunch of new name/brand entries for things in the OSM database when name tags already exist for them just because Wikipedia might use a different name. Its just redundant, not how people tag, and over complicates things.

I agree with this too. I wouldn't override either the mapper's preferences or the brand's preferences unless we have one of the few good reasons to mentioned above. 

> There's also nothing wrong with having multiple entries for the same company if need be. I

Yes!  We already do this for international brands too.  Separate entries for "Starbucks", "Старбакс", "星巴克", because that's how they are branded in real life, and what it should say on maps.

> (maybe we could open a meta issue or something for ones that we do think should changed. In general there's nothing wrong with opening issues for things that we are unsure about and also getting feedback from other contributors/admins about it).

💯 always ok to open up another issue to discuss or talk it through...

**Comment by matkoniecz** [comment]
I want to add that `name` tag, as usual, is the common name used for the object.

> We almost always want to use the name that the brand prefers

works because usually companies have both significant influence on name used to refer to them and because typical company is selecting names that can be used by people.

So usually, common name, name used by company and signposted name is the same.

Though there are cases where people are commonly using shortened versions ("Applebee’s Neighborhood Grill & Bar" vs "Applebee's"), sometimes even overriding what is signposted ("Starbucks Coffee" vs "Starbucks"). In such cases we can and should have `name` tag differing from official brand name.

**Comment by jnicho02** [comment]
My take is that you have to involve the local community. They are the ones who will care about the finer details in their area...and they are the ones who will get upset if they think that stuff gets decided without their input. The presets on the iD Editor are very powerful, and with great power comes great responsibility. I'm not saying that everything has to be discussed on a tagging mailing list and voted for...but for some decisions we need to review existing local tagging (which could be out-of-date and/or misguided) and ask questions on a local mailing list to get input.

When it comes to brands, the UK community is fairly organised and have an osmwiki page https://wiki.openstreetmap.org/wiki/Retail_chains_in_the_United_Kingdom I know that @SilentSpike is using this page to drive edits to the NSI, which is great. I am going to make a point of telling the UK community that this is how they influence the brands in iD/Vespucci and that they are being listened to. Problematic "gb" brands can have a question asked on Talk-GB.
